.travis.yml

```diff
@@ -40,9 +40,8 @@ services:
   - redis
 
 env:
-  - REQUIREMENTS=baseline SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/invenio"
-  - REQUIREMENTS=release SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/invenio"
-  - REQUIREMENTS=devel SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/invenio"
+  - REQUIREMENTS=baseline SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/zenodo"
+  - REQUIREMENTS=baseline SQLALCHEMY_DATABASE_URI="sqlite:///tmp/zenodo.db"
```

Comment:
Where exactly should I add those 2 lines ?
Neither zenodo's nor invenio's .travis.yml file has those lines.